libfreerdp/core/gateway/rpc_client.c

```diff
@@ -425,7 +430,7 @@ int rpc_client_recv_fragment(rdpRpc* rpc, wStream* fragment)
 		pdu->Flags = 0;
 		pdu->Type = header->common.ptype;
 		pdu->CallId = header->common.call_id;
-		if (Stream_EnsureCapacity(pdu->s, Stream_Length(fragment)))
```

Comment:
That one seems to have slipped thorough in my booleanize Stream_EnsureCapacity commit. Good catch. Thank you.